extra/2-piping.js

```diff
@@ -16,26 +16,30 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a,b) {
+return a+b;
 }
 
-function multiply() {
-
+function multiply(a,b) {
+return a*b;
 }
-
-function format() {
-
+function format(a) {
+  return `£${a}`;
 }
 
 const startingValue = 2;
 
-// Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+// Why can this code be seen as bad practice? Comment your answer. 
+//this is doing quite a lot on one line. Adding an extra line with a variable make this more clear.
+
+let badCode ="£"+((startingValue+10)*2);
```

Comment:
You could also use the functions for this too:
`let badCode = format(multiply(add(startingValue,10),2))`
(btw this is still bad practice, what you did below is better)